python/pylibwholegraph/pyproject.toml

```diff
@@ -27,15 +27,11 @@ authors = [
     { name = "NVIDIA Corporation" },
 ]
 license = { text = "Apache 2.0" }
-requires-python = ">=3.6"
+requires-python = ">=3.10"
```

Comment:
This code is living in 2 places right now.

I've asked if keeping a floor this old was intentional over here: https://github.com/rapidsai/cugraph-gnn/pull/32#discussion_r1727298652

Reply:
Update from that thread: It was not intentional. We are safe to update this.